packages/superset-ui-demo/storybook/stories/mocks/formData.js

```diff
@@ -0,0 +1,35 @@
+/* eslint sort-keys: 'off' */
+/** The form data defined here is based on default visualizations packaged with Apache Superset */
+
+export const bigNumberFormData = {
```

Comment:
perhaps separate this into multiple files to avoid this growing into one file with all formdata

Reply:
👍 I'm adding some more in the other branch, so I'll separate then if that's okay